### DISCUSSION TITLE
同一网络，Windows能看见手机，手机找不到PC

### DISCUSSION THREAD

**Discussion by jason-shcn** [localsend/localsend#2166] [L-简体中文]
同一网络，Windows App打开能看见手机，手机找不到PC。
Windows环境1.16.1，手机是iOS18.2，app版本1.16.2。
windows升级了火绒后就这样了，请问下要对哪个端口或者服务进行操作吗？

**Comment by sfcboy** [comment]
windows安全中心-防火墙和网络保护-高级设置-最右边一列的单击属性-专用/公用配置文件-入站连接改为允许
